### ISSUE TITLE
RouterModule.resolvePath does not concat paths correctly

### ISSUE THREAD

**Issue by jeremygauchel** [nestjsx/nest-router#41]
If controller path does not start with a '/', resolvePath of RouterModule does not work for middlewares.
Example:
```
@Controller('foo') // No '/' at the beginning!
export class FooController {
  @Get('hoo')
  doHoo() { return 'hoo' }
  ...
}

@Injectable()
export class FooMiddleware implements NestMiddleware {
  resolve(...args: any[]): MiddlewareFunction {
    return (req, res, next) => {
      console.log('this message will not be logged... so sad..');

      next();
    };
  }
}

@Module(...)
FooModule {
  configure(consumer: MiddlewareConsumer) {
    consumer.apply(FooMiddleware).forRoutes(
      RouterModule.resolvePath(FooController ),
    );
  }
}
```

So may proposal is to do something like:

```
  public static resolvePath(controller: Type<Controller>): string {
    const controllerPath: string = Reflect.getMetadata(PATH_METADATA, controller);
    const modulePath = RouterModule.routesContainer.get(controller.name);
    if (modulePath && controllerPath) {
      return validatePath(modulePath + (!controllerPath.startsWith('/') ? '/' : '') + controllerPath);
    } else {
      throw new UnknownElementException();
    }
  }
```

**Comment by shekohex** [comment]
ok the bug is here
https://github.com/shekohex/nest-router/blob/d7fae4f2b8b906bedaa737390ba8b4201853256d/src/router.module.ts#L46
i'm sure, it would be fixed if we used that instead.
```js
return validatePath(modulePath + validatePath(controllerPath));
```
@jeremygauchel  feel free to create a new PR, and kindly add at least one `test` for that issue, so we can make sure that everything is works as expected.

**Comment by shekohex** [comment]
fixed in `v1.0.9` :tada:
